### ISSUE TITLE
RCS0036 does not report blank lines between single-line properties in records

### ISSUE THREAD

**Issue by Sumits-Cosmos** [JosefPihrt/Roslynator#1812]
### Summary

RCS0036 correctly reports a blank line between consecutive single-line property declarations when they are declared inside a class, but does not report the same pattern when the properties are declared inside a record.  The same behavior is reproducible with Roslynator.Formatting.Analyzers 4.16.0.

### Rule ID (if applicable)

RCS0036

### Description

RCS0036 (`Remove blank line between single-line declarations of same kind`)
reports the expected diagnostic when consecutive single-line properties are
declared inside a `class`.

However, the same declarations inside a `record` do not produce an RCS0036
diagnostic.

### Environment

- Roslynator.Formatting.Analyzers: 4.16.0
- .NET SDK: [your version]
- C#: [your version, if known]
- OS: Ubuntu [your version]
- Formatter: `dotnet format analyzers`
- Diagnostic: RCS0036

### Expected behavior

RCS0036 should report the blank line in both cases because the declarations
are the same kind of single-line declarations.

### Actual behavior

RCS0036 reports the diagnostic for the `class` case but does not report it for
the equivalent `record` case.

#### Case 1 - class

```csharp
public class Test
{
    public string A { get; set; }

    public string B { get; set; }
}
```
Running:
`
dotnet format analyzers --diagnostics RCS0036 --severity info
`
reports:
RCS0036: Remove blank line between single-line declarations of same kind

Case 2 - record

```csharp
public record Test
{
    public string A { get; set; }

    public string B { get; set; }
}
```
Running the same command:

`dotnet format analyzers --diagnostics RCS0036 --severity info`

does not report RCS0036.
